### PR TITLE
Update codespaces docs

### LINKED ISSUE #599: GitHub Codespace Documentation
Update Helium readme.md documentation:
- [ ] Remove VS Code Codespaces 
- [ ] Update to reflect GitHub Codespace guidance

### PR DESCRIPTION
**Pull request by bartr**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- update docs to use github codespaces

## Issues Closed or Referenced

- Closes #599

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +58,7 @@ Setup using Visual Studio Codespaces
     - choose view
     - choose terminal
 
-![alt text](./docs/images/codespaces4.jpg "Codespaces Menu Button")
+![Codespaces Menu Button](./docs/images/codespaces4.jpg "Codespaces Menu Button")
 
 > If using Codespaces, skip to the Azure login step
```
Comment (sivamu) [inline README.md:63]: If dotfiles are not enabled for github codespaces, should this step still be skipped? I suppose it'll be added soon but for the time being, the login will be needed.

**Review by sivamu — APPROVED** [review]
Approved with a minor comment

File: README.md (hunk 1 above)
Comment (bartr) [reply README.md:63]: the step we're skipping is git clone - since you're in codespaces, the workspace is already setup

Comment (sivamu) [reply README.md:63]: ah can't read

**Pull request merged**